interface/MaxLikelihoodFit.h

```diff
@@ -70,7 +70,8 @@ class MaxLikelihoodFit : public FitterAlgoBase {
     std::string channel;
     RooArgList  obs;
     const RooAbsReal *norm;
-    const RooAbsPdf  *pdf;
+    const RooAbsReal  *pdf;
```

Comment:
Saving the shapes has to work a bit differently when we have a RooRealSumPdf instead of a RooAddPdf. Here I changed RooAbsPdf to RooAbsReal we can take a pointer to a pdf or a function.